presentation/TASKER_Presentation.md

```
@@ -928,6 +932,77 @@ tasker/
 
 ---
 
+# NEW in v2.1: Config-Based Execution Types
+
+### Flexible, Extensible Execution Architecture
```

markdownlint-cli2 (0.18.1)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)